include/aws/common/package.h

```diff
@@ -0,0 +1,23 @@
+#ifndef AWS_COMMON_PACKAGE_H
+#define AWS_COMMON_PACKAGE_H
+
+/*
+ * Copyright 2010-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+#define AWS_PACKAGE_SLOTS 16
```

Comment:
What does this do? I'll probably find out later in the PR, but it should probably have a comment.